src/platform/keyboard_input.c

```diff
@@ -74,8 +74,9 @@ void platform_handle_key_down(SDL_KeyboardEvent *event)
     switch (event->keysym.sym) {
         case SDLK_RETURN:
         case SDLK_KP_ENTER:
-            keyboard_return();
-            hotkey_enter();
+            if(!is_alt_down(event))
+                keyboard_return();
```

Comment:
Formatting: space after `if` and always use curly braces, even though it's only one statement, please.

Functional: why should Alt+Enter not be recognised as an enter on the keyboard? In other words, why would we need the if at all?

Reply:
Do you want me to move the check inside `keyboard_return()`? None of the `keyboard_*` functions seemed to do any logic, which is why I didn't put the check in there.

If we remove the if completely, then pressing <kbd>Alt</kbd> <kbd>Return</kbd> to switch to full screen will also confirm some dialogs. I think that this is an unwanted behavior.

For example:
https://github.com/bvschaik/julius/blob/8076f54afa7107ceae177759e8a46a41758e0411/src/window/new_career.c#L78

Reply:
> If we remove the if completely, then pressing Alt Return to switch to full screen will also confirm some dialogs. I think that this is an unwanted behavior.

Fair point. It's ok like this then :)

Reply:
By the way, do you use any specific coding style? It would be nice to add a `.clang-format` file at the root of the repository so that IDEs such as VSCode and CLion can use it to automatically format the code.

There are some built-in coding styles, and you can build your own interactively here: https://zed0.co.uk/clang-format-configurator/ (you can use as a base the style that is the closest to your rules).